pkg/translator/api/defaults.go

```diff
@@ -11,9 +11,10 @@ var (
 	DefaultEdgeLBPoolCreationStrategy = EdgeLBPoolCreationStrategyIfNotPresent
```

Comment:
These could all be consts too, right?

Reply:
I'll keep it for now and change it in a follow up PR.